k-distribution/include/kframework/builtin/domains.md

```diff
@@ -1027,6 +1030,7 @@ You can get the number of elements of a list in O(1) time.
 
 ```k
   syntax Int ::= size(List)               [function, total, hook(LIST.size), symbol(sizeList), smtlib(smt_seq_len)]
+  syntax {Width} MInt{Width} ::= size(List) [function, hook(LIST.sizeMInt), symbol(sizeMInt)]
```

Comment:
Note that this is a partial function, not defined for sizes greater than 2^width.